backend/app/server.py

```diff
@@ -20,7 +20,7 @@
 @asynccontextmanager
 async def lifespan(_: FastAPI):
     log.info("Starting up...")
-    models.run_migrations()
+    # models.run_migrations()
```

Comment:
Uncomment this line for now, we should have a separate PR to handle updating the migration steps.